flink-connector-jdbc/pom.xml

```diff
@@ -110,6 +110,13 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 
+        <dependency>
+            <groupId>com.oceanbase</groupId>
+            <artifactId>oceanbase-client</artifactId>
```

Comment:
The Java client here https://github.com/oceanbase/obconnector-j seems to have a LGPL License. This is not a permissive licence - so it seems we should not ship this until there is an acceptable license - such as Apache 2.0.

Reply:
It indeed uses LGPL license. 

I'm not familiar with the legal issues, is it forbidden to include a LGPL dependency with 'provide' scope?

Reply:
The Apache policy is explained at https://www.apache.org/legal/resolved.html. You can't include source or binary code for LGPL licensed work.

Reply:
> The Apache policy is explained at https://www.apache.org/legal/resolved.html. You can't include source or binary code for LGPL licensed work.

I checked the jira list, and it seems that it's OK to include LGPL dependency when it's optional and not included in the dist package. Am I understanding correctly? https://issues.apache.org/jira/browse/LEGAL-477